core-web/libs/edit-content/src/lib/fields/dot-edit-content-radio-field/dot-edit-content-radio-field.component.spec.ts

```diff
@@ -0,0 +1,233 @@
+import { Spectator, createComponentFactory } from '@ngneat/spectator';
+
+import { ControlContainer, FormGroupDirective } from '@angular/forms';
+
+import { RadioButton } from 'primeng/radiobutton';
+
+import { DotEditContentRadioFieldComponent } from './dot-edit-content-radio-field.component';
+
+import {
+    RADIO_FIELD_BOOLEAN_MOCK,
+    RADIO_FIELD_FLOAT_MOCK,
+    RADIO_FIELD_INTEGER_MOCK,
+    RADIO_FIELD_TEXT_MOCK,
+    createFormGroupDirectiveMock
+} from '../../utils/mocks';
+
+describe('DotEditContentRadioFieldComponent', () => {
+    let spectator: Spectator<DotEditContentRadioFieldComponent>;
+
+    const createComponent = createComponentFactory({
+        component: DotEditContentRadioFieldComponent,
+        componentViewProviders: [
+            { provide: ControlContainer, useValue: createFormGroupDirectiveMock() }
+        ],
+        providers: [FormGroupDirective],
+        detectChanges: false
+    });
+
+    beforeEach(() => {
+        spectator = createComponent();
+    });
+    it('should if have an object without value and label, the label must be the same as the value, and of the same type', () => {
```

Comment:
```suggestion
    it('should set the key/value the same when bad formatting options passed', () => {
```

The same we added to select.